src/index.js

```diff
@@ -115,6 +115,8 @@ function AceDiff(options) {
   this.editors.right.ace.setReadOnly(!this.options.right.editable);
   this.editors.left.ace.setTheme(getTheme(this, C.EDITOR_LEFT));
   this.editors.right.ace.setTheme(getTheme(this, C.EDITOR_RIGHT));
+  this.editors.left.ace.$blockScrolling = Infinity;
```

Comment:
Could you also add a code comment to these lines so it's clear what they are doing - possibly also linking to some Ace docs or something

Reply:
done